hpat/compiler.py

```diff
@@ -150,145 +63,104 @@ def inline_calls(func_ir, _locals):
     # CFG simplification fixes this case
     func_ir.blocks = ir_utils.simplify_CFG(func_ir.blocks)
 
+#TODO: remove these helper functions when Numba provide appropriate way to manipulate passes
+def pass_position(pm, location):
+    assert pm.passes
+    pm._validate_pass(location)
+    for idx, (x, _) in enumerate(pm.passes):
+        if x == location:
+            return idx
+    else:
```

Comment:
`else:` is excess and could be removed because we don't have `break` in the loop. Let's leave only `raise`.

Reply:
fixed